ibis/pandas/execution/selection.py

```diff
@@ -277,7 +279,9 @@ def physical_tables_node(node):
 
 
 @execute_node.register(ops.Selection, pd.DataFrame)
-def execute_selection_dataframe(op, data, scope=None, **kwargs):
+def execute_selection_dataframe(
+    op, data, scope=None, timecontext=None, **kwargs
```

Comment:
Can you remove the default for scope and timecontext here?

Reply:
+1 done